doc/edge-metrics-v2.md

```diff
@@ -0,0 +1,43 @@
+|------------------------------------------|--------|
+| `ExperimentalFeatures__Enabled`          | `true` |
+| `ExperimentalFeatures__EnableMetrics`    | `true` |
+
+### Metrics
+
+| Name                                                        | Dimensions                                                                                                                                                                           | Description                                                                                                                                                                                              | Type    |
+|-------------------------------------------------------------|--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|---------|
+| `edgehub_gettwin_total`                                     | `iothub` (IoT Hub Name)<br> `edge_device` (IoT Edge device name)<br> `source` (Operation source)<br> `id` (Module ID)                                                                | Get Twin calls                                                                                                                                                                                           | counter |
+| `edgehub_messages_received_total`                           | `iothub` (IoT Hub Name)<br> `edge_device` (IoT Edge device name)<br>  `protocol` (Protocol received on)<br> `id` (Module ID)                                                         | Messages received from clients                                                                                                                                                                           | counter |
+| `edgehub_messages_sent_total`                               | `iothub` (IoT Hub Name)<br> `edge_device` (IoT Edge device name)<br> `from` (Message source)<br> `to` (Message destination)                                                          | Messages sent to clients/upstream                                                                                                                                                                        | counter |
+| `edgehub_message_size_bytes_sum`                            | `iothub` (IoT Hub Name)<br> `edge_device` (IoT Edge device name)<br> `id` (Module ID)                                                                                                | Sum of bytes received from clients                                                                                                                                                                       | summary |
```

Comment:
I think this is just edgehub_message_size_bytes as a summary, for which prometheus emits sum and count.
Also instead of "bytes received from clients", it should be "Bytes received as messages" or "Sum of size of messages" or something like that (It does not count bytes received over other primitives)